portfolio/src/main/webapp/index.html

```diff
@@ -5,63 +5,65 @@
     <!-- Imported Libraries: Google's Material Design Library and Google's Fonts Library -->
     <link href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css" rel="stylesheet">
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```

Comment:
How is jquery used in this PR?

Reply:
That is an excellent point-it is not 🤦 I will remove that-thanks for pointing it out!